torchrl/objectives/ppo.py

```diff
@@ -652,7 +722,34 @@ def forward(self, tensordict: TensorDictBase) -> TensorDict:
             loss_critic = self.loss_critic(tensordict)
             td_out.set("loss_critic", loss_critic.mean())
 
+        if self.kl_coef:
+            loss_kl_init = self.kl_coef * self._kl_init(dist, tensordict)
+            td_out.set("loss_kl_init", loss_kl_init.mean())
```

Comment:
I will probably change also this to `kl_loss` or `loss_kl`

Reply:
kl_loss no because we need all losses to start by loss by convention
For the KLPENPPOLoss we already have a non-differentiable "kl" entry so it's confusing if we have loss_kl and kl together, one would naturally expect them to be related when they're not.
Maybe `"loss_actor_shift"`?

Reply:
"loss_kl_actor_shift" is nice but my concern is that we have 2 losses that are based on KL. we need to make sure it's clear its the kl computed against:
- ref/initial/starting model
- previous step/iteration model

so would change "kl" entry also

Reply:
kl_to_init
kl_to_prev

Reply:
kl_wrt_init
kl_wrt_prev
?
otherwise good your version with \_to\_